.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+**Do you want to request a *feature* or report a *bug*?**
```

Comment:
Or ask a question.

Reply:
If it's a question probably should go to Discord?

Reply:
Ideally, but if GitHub is their favorite mean of communication then they should feel welcome to post questions.

Reply:
I'd rather not encourage that 😄